src/search/search_algorithms/iterated_search.cc

```diff
@@ -185,6 +183,18 @@ class IteratedSearchFeature
             "(using heuristic predefinition) between iterations, "
             "the path data (that is, landmark status for each visited state) "
             "will be saved between iterations.");
+        document_note(
+            "Semantics of Search Bounds",
+            "The definition of our interfaces for search algorithms allow to "
+            "set bounds for the iterated search and all its component search "
```

Comment:
This construction with "allow to" is apparently a typical English language mistake that native speakers of German do. Chris Beck pointed this out in the context of Florian's thesis.

Reply:
I guess I'm also susceptible to the mistake, so I'm not 100% sure about the following. But I think the error is that "allow" requires saying who is allowed something, i.e., you need a direct object like "allow the users to ...". The better solution is almost always to use a different phrasing.